ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -166,20 +160,16 @@ private void applyRAs(Crac crac, Network network, List<RemedialActionResult> raR
 
     private double getMinMargin(Crac crac, SystematicSensitivityAnalysisResult systematicSensitivityAnalysisResult) {
         double minMargin = Double.POSITIVE_INFINITY;
-        try {
-            for (Cnec cnec : crac.getCnecs()) {
-                double margin;
-                double flow = systematicSensitivityAnalysisResult.getCnecFlowMap().getOrDefault(cnec, Double.NaN);
-                double margin1 = cnec.getThreshold().getMaxThreshold(Unit.MEGAWATT).orElse(Double.POSITIVE_INFINITY) - flow;
-                double margin2 = flow - cnec.getThreshold().getMinThreshold(Unit.MEGAWATT).orElse(Double.POSITIVE_INFINITY);
-                margin = Math.min(margin1, margin2);
-                if (Double.isNaN(margin)) {
-                    throw new FaraoException(format("Cnec %s is not present in the linear RAO result. Bad behaviour.", cnec.getId()));
-                }
-                minMargin = Math.min(minMargin, margin);
+        for (Cnec cnec : crac.getCnecs()) {
+            double margin;
+            double flow = systematicSensitivityAnalysisResult.getCnecFlowMap().getOrDefault(cnec, Double.NaN);
+            double margin1 = cnec.getThreshold().getMaxThreshold(Unit.MEGAWATT).orElse(Double.POSITIVE_INFINITY) - flow;
+            double margin2 = flow - cnec.getThreshold().getMinThreshold(Unit.MEGAWATT).orElse(Double.POSITIVE_INFINITY);
```

Comment:
should probably be orElse(Double.NEGATIVE_INFINITY) for margin2 instead

Reply:
It is not related to my dev but ok I changed it